### ISSUE TITLE
assumes that your contact email will be the same as your login email

### ISSUE THREAD

**Issue by ejheil** [TobiasTheViking/flickrannex#1]
If your contact email for your flickr account is different from your login email, it will fail to "find" your account and will fail during setup.

**Comment by TobiasTheViking** [comment]
Besides for a notification on setup "Your login email address must be your primary or secondary address", is there anything i can do?

How did you fix it? Did you just change email settings no flickr?

**Comment by ejheil** [comment]
I changed my email settings on flickr.  I put in my yahoo login the first time, when it took me to the URL where I authorized it, and that worked.  Then it failed until I changed my notification email to match my yahoo login.

Now that I think about it, I'm not sure it would have been a problem if I just used my notification email in the first place.  I can experiment with that (after de-authorizing the app) and report back.

**Comment by ejheil** [comment]
To clarify, my assumption when I filed the bug report was that there needed to be, potentially, two different addresses:  one for authorizing the app, one used later on when the app does a "find user by email address" for some reason. 

It's possible I could have used the contact ID for both, and everything would have been fine.  I'd like to check that possibility.

**Comment by ejheil** [comment]
OK, I went through this again and the problem was really on my end.  My yahoo _email address_ wasn't associated at all with my flickr account, even though of course I had to use my yahoo account to log into flicker.

When I entered into flickrannex an email address which was associated with flickr, everything was fine.

So the problem was really me assuming that my yahoo email address was relevant to my flickr account when it wasn't.

I guess this is partly just me not knowing what was going on, partly the weirdness that you must necessarily have a _yahoo account_ associated with flickr but might not necessarily have your _yahoo email address_ associated with your flickr account....  

It might be helpful to have an FAQ entry for this situation, but the program seems to be behaving fine, it was just my weird situation that caused the problem.
